fms/models/hf/mixtral/__init__.py

```diff
@@ -49,23 +49,27 @@ def convert_to_hf(
         )
         for i, oss_hf_layer in enumerate(oss_hf_model.model.layers):
             fms_hf_layer = fms_hf_model.decoder.model.layers[i]
+            hf_q, hf_k, hf_v = torch.split(
+                fms_hf_layer.attn.in_proj.qkv_fused.weight,
+                fms_hf_layer.attn.in_proj.splits,
+            )
 
             # self attn (+ HF RoPE transpose)
-            hf_q = fms_hf_layer.attn.query.weight
+            hf_q = hf_q
```

Comment:
we don't really need this line and `hf_k = hf_k`

Reply:
fixed in latest commit